### ISSUE TITLE
Searching for special characters like <,>,= fails in list.js search.

### ISSUE THREAD

**Issue by runwaldarshu** [javve/list.js#147]
Instead of actual characters lt; gt; is used it works.

**Comment by simonseddon** [comment]
Has this / can it be implemented. Not being able to search for special characters is a deal-breaker for so many uses, including my list of songs (where I'm, I've, Love's, You're, You & Me, etc. are all the rage).

**Comment by BenjaminEHowe** [comment]
I'm also hitting this issue, it's somewhat annoying that I can't use the CDN...

**Comment by simonseddon** [comment]
See issue #327 for a solution which requires no modification to the script. It does however mean that special characters aren't searchable - as they're stripped out of both the search input AND searchable objects. Worked great for my use case.

**Comment by cbier** [comment]
@javve this hasn't been implemented into the main library yet, right?

**Comment by wolffe** [comment]
No, list.js seems dead. I need it badly for one of my projects, but it seems I would need to fork it and add all the current pull requests.

**Comment by cbier** [comment]
I am just modifying the library locally to add the things I need, maybe that will be a quicker solution for you @wolffe . For instance, these changes I am having to do myself.

**Comment by wolffe** [comment]
I have added lots of changes myself, but I would appreciate an official library.

**Comment by cbier** [comment]
Yeah I definitely agree. I am grateful for this library but it's shortcomings seem so big sometimes. There are also great pull requests that have been waiting to be merged in.

I just hope that @javve is ok. Maybe he's having a difficult life situation right now.

Thanks again for the library @javve

**Comment by javve** [comment]
Wow, heh, I'm really sorry for my lack of updates around List.js :cold_sweat: 
...and thanks a lot for the concern @cbier :relaxed: 

I've been working with v1.2.0 for a while now and it's getting closer!

I tried to reproduce this error but I don't think I understand everything exactly. Here is a Codepen I made that includes ", ' and &. http://codepen.io/javve/pen/xVxZrw

For me it works if I search with for " and '.
& works if I write `Martina&` but not if I write something after &. Weird.

How do you want it to work?

**Comment by BenjaminEHowe** [comment]
On that codepen, searching for tina&E should yield the obvious result :-)

Chrome 48 on Windows 10 Pro

**Comment by Er-Kalpesh** [comment]
If I have any text like "A&B" and search with following cases

1) Search A& : Works
2) Search & : Works
3) Search : A&B : Not working
4) Search &B : Not working

So how to fix aboave issue ?

**Comment by avveshka** [comment]
in my case i changed 
`setSearchString: function (e) { e = (e = t.utils.toString(e).toLowerCase()).replace(/[-[\]{}()*+?.,\\^$|#]/g, **"\\$&"**)`
to
`setSearchString: function (e) { e = (e = t.utils.toString(e).toLowerCase()).replace(/[-[\]{}()*+?.,\\^$|#]/g, **"\$&"**)`
in min bundle. And now it works
